Zilon.Core/Zilon.Core.Benchmarks.CreateSector/CreateCellularAutomatonSectorBench.cs

```diff
@@ -0,0 +1,72 @@
+﻿using BenchmarkDotNet.Attributes;
+
+using JetBrains.Annotations;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Zilon.Core.Client;
+using Zilon.Core.Graphs;
+using Zilon.Core.Persons;
+using Zilon.Core.Players;
+using Zilon.Core.Schemes;
+using Zilon.Core.Tactics;
+using Zilon.Core.Tests.Common;
+
+namespace Zilon.Core.Benchmarks.CreateSector
+{
+    public class CreateCellularAutomatonSectorBench
+    {
+        private ServiceProvider _serviceProvider;
+
+        [Benchmark(Description = "Create CA Sector")]
+        public async System.Threading.Tasks.Task CreateSectorAsync()
+        {
+            var sectorManager = _serviceProvider.GetRequiredService<ISectorManager>();
+
+            await sectorManager.CreateSectorAsync().ConfigureAwait(false);
+        }
+
+
+        [IterationSetup]
+        public void IterationSetup()
+        {
+            var startUp = new Startup();
+            var serviceCollection = new ServiceCollection();
+            startUp.RegisterServices(serviceCollection);
+
+            _serviceProvider = serviceCollection.BuildServiceProvider();
+        }
+
+        private IActorViewModel CreateHumanActorVm([NotNull] IPlayer player,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CreateHumanActorVm'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310821)